docs/advanced_topics/embeds.md

```diff
@@ -405,3 +405,6 @@ repopulate the records that are being used on the site.
 You may want to do this if you've changed from oEmbed to Embedly or vice-versa
 as the embed code they generate may be slightly different and lead to
 inconsistency on your site.
+
+In general whenever you make changes to embed settings you are required to clear out 
+Embed objects using [purge_embeds command](purge_embeds).
```

Comment:
This can probably be on the same line as above. 

I would not say 'required' but rather it is recommended that you...